booking/manager.py

```diff
@@ -0,0 +1,47 @@
+from booking.storage import get_storage
+from booking.models import Event, TicketOrder
+
+
+class DataManager:
+
+    def __init__(self):
+        self._storage = get_storage()
+
+    def can_buy_tickets(self, event_id):
+        orders = self._storage.get_orders_for_event(event_id)
+        active_orders = filter(lambda order: order.activate, orders)
+        tickets_sold = sum(order.ticket_count for order in active_orders)
+        total_tickets = self._storage._events.get(event_id).total_ticket_count
+        return total_tickets - tickets_sold
```

Comment:
This function is not used anywhere else but in tests, probably it should be turned into private function and used in function below to fix `# TODO check availability first!`